packages/web-config-server/src/apiV1/dashboardses.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { RouteHandler } from './RouteHandler';
+import { NoPermissionRequiredChecker } from './permissions';
+
+export default class extends RouteHandler {
+  static PermissionsChecker = NoPermissionRequiredChecker;
+
+  buildResponse = async () => {
+    const { entity, query } = this;
+    const { code: entityCode } = entity;
+    const userGroups = await this.req.getUserGroups(entityCode);
+    const hierarchyId = await this.fetchHierarchyId();
+    const returnArray = [];
+    // Fetch dashboards
+    const dashboards = await this.models.dashboard.getDashboards(
+      entity,
+      hierarchyId,
+      query.projectCode,
+    );
+    // Fetch dashboard items
+    await Promise.all(
```

Comment:
```suggestion
    const returnArray = await Promise.all(
```
And then you can just return inside the map function rather than assign

Reply:
Also, descriptive variable names ftw
```suggestion
    const dashboardsAndItems = await Promise.all(
```